File: drs/src/core/com/db4o/drs/inside/TestableReplicationProviderInside.java
/* Copyright (C) 2004 - 2005  db4objects Inc.  http://www.db4o.com */

package com.db4o.drs.inside;

public interface TestableReplicationProviderInside extends ReplicationProviderInside, SimpleObjectContainer {

	boolean supportsMultiDimensionalArrays();

	boolean supportsHybridCollection();

	boolean supportsRollback();

	boolean supportsCascadeDelete();
}

File: drs/drs.net/drs/src/com/db4o/drs/inside/TestableReplicationProviderInside.cs
namespace com.db4o.drs.inside
{
	public interface TestableReplicationProviderInside : com.db4o.drs.inside.ReplicationProviderInside
		, com.db4o.drs.inside.SimpleObjectContainer
	{
		bool SupportsMultiDimensionalArrays();

		bool SupportsHybridCollection();

		bool SupportsRollback();

		bool SupportsCascadeDelete();
	}
}
